java/wfs-xml-query/README.md

```diff
@@ -2,17 +2,21 @@
 
 Load a WFS feature table using an XML query.
 
-![Load WFS with XML query App](wfs-xml-query.png)
+![Image of load WFS with XML query](wfs-xml-query.png)
 
 ## Use case
 
 Runtime `QueryParameters` objects can't represent all possible queries that can be made against a WFS feature service. For example, Runtime query parameters don't support wildcard searches. You can provide queries as raw XML strings, allowing you to access query functionality not available with `QueryParameters`.
 
+## How to use the sample
+
+Run the sample and view the data loaded from the the WFS feature table.
+
 ## How it works
 
 1. Create a `WfsFeatureTable` and a `FeatureLayer` to visualize the table.
-1. Set the feature request mode to `ManualCache`. 
-1. Call `PopulateFromServiceWithXmlAsync` to populate the table with only those features returned by the XML query.
+2. Set the feature request mode to `ManualCache`. 
```

Comment:
`statesTable.setAxisOrder(OgcAxisOrder.NO_SWAP);`

Is it important to note that the axis order is "no swap"? This step seems a bit mystifying to me, it might be useful to state why this is necessary. Perhaps this belongs in an "about the data" or "additional info" section.

Reply:
i think this is ok, we have other samples that use it and we don't explain why: there is API ref that explains it in more detail and I think someone using these WFS services will be familiar with what and why they should set the axis order as.